content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -21,6 +21,18 @@ deprecated API versions to newer and more stable API versions.
 ## Removed APIs by release
 
 
+### v1.27
+
+The **v1.27** release will stop serving the following deprecated API versions:
+
+#### HorizontalPodAutoscaler {#horizontalpodautoscaler-v127}
+
+The **autoscaling/v2beta2** API version of HorizontalPodAutoscaler will no longer be served in v1.27.
+
+* Migrate manifests and API clients to use the **autoscaling/v2** API version, available since v1.23.
+* All existing persisted objects are accessible via the new API
+* No notable changes
```

Comment:
@reylejano if you omit these two lines
```markdown
* All existing persisted objects are accessible via the new API
* No notable changes
```
then we can merge as-is and track a follow-up PR to actually document what the migration advice is. I think that might help to move things forward.